ts/Core/Chart/MapChart.ts

```diff
@@ -49,6 +49,14 @@ declare module './ChartLike'{
  * @extends Highcharts.Chart
  */
 class MapChart extends Chart {
+    /**
+     * The map view handles zooming and centering on the map, and various
+     * client-side projection capabilities.
+     *
+     * @name Highcharts.MapChart#mapView
+     * @type {Highcharts.MapView|undefined}
+     */
+
```

Comment:
This should be over the assignment. If JSDoc does not pick it up, then move the doclet to the end of `MapView.ts` followed by `'';`.

Reply:
I mean this assignment: https://github.com/highcharts/highcharts/blob/master/ts/Maps/MapView.ts#L1374